databox/api/src/Integration/Phrasea/Expose/ExposeClient.php

```diff
@@ -207,4 +254,21 @@ public function deleteAsset(IntegrationConfig $config, IntegrationToken $integra
             ->request('DELETE', '/assets/'.$assetId)
         ;
     }
+
+    private function putPart(string $url, mixed $handleFile, int $partSize, int $retryCount)
+    {
+        if ($retryCount > 0) {
+            $retryCount--;
+            try {
+                return $this->uploadClient->request('PUT', $url, [
+                    'body' => fread($handleFile, $partSize),
+                ])->getHeaders();
```

Comment:
Still TODO: try to use closure

Reply:
fread always return false on the body closure